node/node.go

```diff
@@ -498,6 +498,9 @@ func NewNode(config *cfg.Config,
 			&pex.PEXReactorConfig{
 				Seeds:    splitAndTrimEmpty(config.P2P.Seeds, ",", " "),
 				SeedMode: config.P2P.SeedMode,
+				// see consensus/reactor.go: blocksToContributeToBecomeGoodPeer
+				// 10000 blocks assuming 1s blocks ~ 2.7 hours.
```

Comment:
> Would be super cool if this could be dynamic based on the actual block latencies from the live network

Thought so too! But it's hard to pull right now since a) `blocksToContributeToBecomeGoodPeer` is private and can be moved/changed in the future b) average block time is not exposed in consensus (prometheus does this, but we can't rely on external services)

Reply:
https://github.com/tendermint/tendermint/issues/3523